source/reference/minio-mc-admin.rst

```diff
@@ -50,11 +50,6 @@ The following table lists :mc:`mc admin` commands:
           :start-after: start-mc-admin-cluster-iam-desc
           :end-before: end-mc-admin-cluster-iam-desc
 
-   * - :mc-cmd:`mc admin console`
-     - .. include:: /reference/minio-mc-admin/mc-admin-console.rst
-          :start-after: start-mc-admin-console-desc
```

Comment:
We might as well clean this fragment if it's no longer being referenced anywhere.

Reply:
Dang it, I thought i caught that one. Thanks.